src/handlers/process-subscriptions.ts

```diff
@@ -59,13 +59,14 @@ export const processSubscriptions = async (
   });
 };
 
-export const handler = async ({
-  providerSponsorSubscriptions,
-  baseLogOptions,
-}: {
+export const handler = async (payload: {
```

Comment:
I played around with the idea of adding the aws context object to the `state` var in psp.ts and then forward it here after doing `lambda.invoke()` but I dropped the idea after seeing that it didn't add much value.